package.json

```diff
@@ -29,31 +29,36 @@
     "test:all": "ember try:each"
   },
   "dependencies": {
-    "ember-cli-babel": "^6.16.0",
+    "ember-cli-babel": "^7.11.1",
     "ember-compatibility-helpers": "^1.1.1"
   },
   "devDependencies": {
-    "@ember/optional-features": "^0.6.3",
-    "ember-cli": "~3.5.0",
-    "ember-cli-dependency-checker": "^3.0.0",
-    "ember-cli-eslint": "^4.2.3",
-    "ember-cli-htmlbars": "^3.0.0",
-    "ember-cli-htmlbars-inline-precompile": "^1.0.3",
-    "ember-cli-inject-live-reload": "^1.8.2",
-    "ember-cli-template-lint": "^1.0.0-beta.1",
+    "@ember/optional-features": "^0.7.0",
+    "babel-eslint": "^10.0.3",
+    "ember-cli": "3.9.0",
+    "ember-cli-dependency-checker": "^3.1.0",
+    "ember-cli-eslint": "^5.1.0",
+    "ember-cli-htmlbars": "^3.1.0",
+    "ember-cli-htmlbars-inline-precompile": "^2.1.0",
+    "ember-cli-inject-live-reload": "^2.0.1",
+    "ember-cli-template-lint": "^1.0.0-beta.3",
     "ember-disable-prototype-extensions": "^1.1.3",
-    "ember-load-initializers": "^1.1.0",
+    "ember-load-initializers": "^2.1.0",
     "ember-maybe-import-regenerator": "^0.1.6",
-    "ember-qunit": "^3.4.1",
-    "ember-resolver": "^5.0.1",
-    "ember-source": "~3.5.0",
+    "ember-qunit": "^4.5.1",
+    "ember-resolver": "^5.3.0",
+    "ember-source": "~3.12.0",
     "ember-source-channel-url": "^1.1.0",
-    "ember-try": "^1.0.0",
-    "eslint-plugin-ember": "^5.2.0",
-    "eslint-plugin-node": "^7.0.1",
+    "ember-try": "^1.2.1",
+    "eslint-plugin-ember": "^6.10.1",
+    "eslint-plugin-node": "^8",
     "lerna-changelog": "^0.8.0",
     "loader.js": "^4.7.0"
   },
+  "resolutions": {
```

Comment:
Let's remove these, the ember-cli-babel bump is a breaking change so we can drop Node 6